WanderLost/WanderLost/Client/Shared/NavMenu.razor

```diff
@@ -23,19 +23,19 @@
     </svg>
   </NavLink>
   <div class="m-header__items">
-    <NavLink href="/Notifications" class="m-header__item" Match="NavLinkMatch.All">Notifications</NavLink>
-    <NavLink href="/PushNotifications" class="m-header__item" Match="NavLinkMatch.All">Push Notifications</NavLink>
-    <NavLink href="/About" class="m-header__item" Match="NavLinkMatch.All">About</NavLink>
+    <NavLink href="/Notifications" class="m-header__item brand-label" Match="NavLinkMatch.All">Notifications</NavLink>
```

Comment:
Shouldn't `class` go before `href`?

Reply:
this way it's easier to see the pages at a glance